Flow.Launcher.Infrastructure/UserSettings/PluginSettings.cs

```diff
@@ -15,6 +15,13 @@ public void UpdatePluginSettings(List<PluginMetadata> metadatas)
                 if (Plugins.ContainsKey(metadata.ID))
                 {
                     var settings = Plugins[metadata.ID];
+
+                    // TODO: Remove. This is one off for 1.2.0 release.
```

Comment:
Just a note here, this is a one off, and will be deleted in the next release. It is needed as we introduced an additional action keyword.

I have also introduced version in plugin settings in the following commit to lay the ground work for handling future plugin meta updates.